public/app/routes/routes.ts

```diff
@@ -344,9 +344,11 @@ export function setupAngularRoutes($routeProvider: route.IRouteProvider, $locati
       },
     })
     .when('/admin/orgs', {
-      templateUrl: 'public/app/features/admin/partials/orgs.html',
-      controller: 'AdminListOrgsCtrl',
-      controllerAs: 'ctrl',
+      template: '<react-container />',
+      resolve: {
+        component: () =>
+          SafeDynamicImport(import(/* webpackChunkName: "AdminListOrgsPage" */ 'app/features/admin/AdminListOrgsPage')),
+      },
```

Comment:
Might also need to add `reloadOnSearch: false` so it doesn't re-render on url query params changes (e.g. when you open/close dashboard search).

Reply:
Is this something that affects more views?

Reply:
Mostly the views that fetch some data from API when they render.

Reply:
Ah, so basically all migrations I did today 😅